python/ray/data/_internal/compute.py

```diff
@@ -368,6 +373,9 @@ def map_block_nosplit(
         ]
         tasks = {w.ready.remote(): w for w in workers}
         tasks_in_flight = collections.defaultdict(int)
+        max_tasks_in_flight_per_actor = (
```

Comment:
Set it in constructor `self.max_tasks_in_flight_per_actor = max_tasks_in_flight_per_actor or DEFAULT_MAX_TASKS_IN_FLIGHT_PER_ACTOR`?

Reply:
That doesn't work since we use this as a config object for the streaming executor later. This code will all be deleted anyways in the future.